pls update for 1.21.1

```
[18:47:23] [Server thread/INFO]: [dynmap] version=1.21.1-3-70dfa46 (MC: 1.21.1)
[18:47:23] [Server thread/ERROR]: [dynmap] Cannot find [Lnet.minecraft.server.BiomeBase;
[18:47:23] [Server thread/ERROR]: [dynmap] Cannot find net.minecraft.server.BiomeStorage
[18:47:23] [Server thread/ERROR]: [dynmap] Unable to find method getHandle for org.bukkit.craftbukkit.CraftChunk
[18:47:23] [Server thread/ERROR]: [dynmap] Unable to find method getValue for com.mojang.authlib.properties.Property
[18:47:23] [Server thread/ERROR]: [dynmap] Error loading org.dynmap.bukkit.helper.BukkitVersionHelperCB
java.lang.reflect.InvocationTargetException: null
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:74) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?]
	at Dynmap-3.7_Beta6.jar/org.dynmap.bukkit.Helper.loadVersionHelper(Helper.java:15) ~[Dynmap-3.7_Beta6.jar:?]
	at Dynmap-3.7_Beta6.jar/org.dynmap.bukkit.Helper.getHelper(Helper.java:103) ~[Dynmap-3.7_Beta6.jar:?]
	at Dynmap-3.7_Beta6.jar/org.dynmap.bukkit.DynmapPlugin.onLoad(DynmapPlugin.java:900) ~[Dynmap-3.7_Beta6.jar:?]
	at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:59) ~[paper-1.21.1.jar:1.21.1-3-70dfa46]
	at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:18) ~[paper-1.21.1.jar:1.21.1-3-70dfa46]
	at io.papermc.paper.plugin.storage.SimpleProviderStorage.enter(SimpleProviderStorage.java:39) ~[paper-1.21.1.jar:1.21.1-3-70dfa46]
	at io.papermc.paper.plugin.entrypoint.LaunchEntryPointHandler.enter(LaunchEntryPointHandler.java:40) ~[paper-1.21.1.jar:1.21.1-3-70dfa46]
	at org.bukkit.craftbukkit.CraftServer.loadPlugins(CraftServer.java:546) ~[paper-1.21.1.jar:1.21.1-3-70dfa46]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:292) ~[paper-1.21.1.jar:1.21.1-3-70dfa46]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1215) ~[paper-1.21.1.jar:1.21.1-3-70dfa46]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:330) ~[paper-1.21.1.jar:1.21.1-3-70dfa46]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.IllegalArgumentException: Error initializing dynmap - bukkit version incompatible!
	at Dynmap-3.7_Beta6.jar/org.dynmap.bukkit.helper.BukkitVersionHelperGeneric.<init>(BukkitVersionHelperGeneric.java:166) ~[Dynmap-3.7_Beta6.jar:?]
	at Dynmap-3.7_Beta6.jar/org.dynmap.bukkit.helper.BukkitVersionHelperCB.<init>(BukkitVersionHelperCB.java:44) ~[Dynmap-3.7_Beta6.jar:?]
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?]
	... 14 more
[18:47:23] [Server thread/INFO]: [dynmap] Mod Support API available
```

--- a/spigot/src/main/java/org/dynmap/bukkit/Helper.java
+++ b/spigot/src/main/java/org/dynmap/bukkit/Helper.java
@@ -40,7 +40,7 @@ public class Helper {
                 Log.info("Loading Glowstone support");
                 BukkitVersionHelper.helper = loadVersionHelper("org.dynmap.bukkit.helper.BukkitVersionHelperGlowstone");
             }
-            else if (v.contains("(MC: 1.21)")) {
+            else if (v.contains("(MC: 1.21)") || v.contains("(MC: 1.21.1)")) {
                 BukkitVersionHelper.helper = loadVersionHelper("org.dynmap.bukkit.helper.v121.BukkitVersionHelperSpigot121");
             }
             else if (v.contains("(MC: 1.20)") || v.contains("(MC: 1.20.1)")) {

--- a/spigot/src/main/java/org/dynmap/bukkit/Helper.java
+++ b/spigot/src/main/java/org/dynmap/bukkit/Helper.java
@@ -40,7 +40,7 @@ public class Helper {
                 Log.info("Loading Glowstone support");
                 BukkitVersionHelper.helper = loadVersionHelper("org.dynmap.bukkit.helper.BukkitVersionHelperGlowstone");
             }
-            else if (v.contains("(MC: 1.21)") || v.contains("(MC: 1.21.1)")) {
+            else if (v.contains("(MC: 1.21")) {
                 BukkitVersionHelper.helper = loadVersionHelper("org.dynmap.bukkit.helper.v121.BukkitVersionHelperSpigot121");
             }
             else if (v.contains("(MC: 1.20)") || v.contains("(MC: 1.20.1)")) {

